ui/ts/pages/nodes.ts

```diff
@@ -322,6 +330,7 @@ module AdminViews {
                 .nonNegativeRate()
                 .title("DDL Statements")
             ).format(d3.format(".1")).title("Schema Changes")
+              .tooltip("The number of DDL statements across all nodes, averaged over a 10 second period.")
```

Comment:
Max or I will add this once we actually do down-sample based on the time span; right now, all time spans are rendered with datapoints in 10 second intervals.